server/src/routes/api/v1/saladsRouter.js

```diff
@@ -7,9 +7,15 @@ const saladsRouter = new express.Router()
 
 saladsRouter.get("/", async (req, res) => {
     try {
-        const salads = await Salad.query()
+        const saladsSansUsers = await Salad.query()
+        const salads = await Promise.all(saladsSansUsers.map( async (salad) => {
+            salad.user = await salad.$relatedQuery("user")
+            salad.user = salad.user.username
```

Comment:
instead of assigning `.user` twice, we can rethink based on the info needed here:
```js
const user = await salad.$relatedQuery("user") // create variable for this since it is not the actual info we want
salad.username = salad.user.username // consider assigning a `username` property because we are not passing along the whole `user` object
```